### PR TITLE
Hotspots / LINDA_fire docs.

### PR DESCRIPTION
**Pull request by vincentiusvin**
## About The Pull Request
Writes some documentation on hotspot itself, perform_exposure, hotspot_expose, and process.

## Why It's Good For The Game
Who let the docs out?

### CODE REVIEW

**Comment by vincentiusvin** [comment]
>doesn't touch the actual code
>fails linting anyway

Will be a moment.

**Review by LemonInTheDark — CHANGES_REQUESTED** [review]
❤️

File: code/modules/atmospherics/environmental/LINDA_fire.dm [hunk 1]
```diff
@@ -8,7 +8,11 @@
 /turf/proc/hotspot_expose(exposed_temperature, exposed_volume, soh = 0)
 	return
 
-
+/** 
+  * Handles the creation of hotspots and initial activation of turfs. 
+  * Setting the conditions for the reaction to actually happen for gasmixtures
```
[the remark this answers is not in the document]
Comment (vincentiusvin) [reply code/modules/atmospherics/environmental/LINDA_fire.dm]: The line below it is supposed to be the same sentence. Comma doesn't seem to fit there.

[the remark this answers is not in the document]
Comment (LemonInTheDark) [reply code/modules/atmospherics/environmental/LINDA_fire.dm]: :)

**Review by LemonInTheDark — APPROVED** [review]

**Comment by Cyberboss** [comment]
Good PR

**Pull request merged**